### PR TITLE
Show instruction on a user's first visit

### PR DESCRIPTION
**Pull request by wilsonych**
Now the  initial status of `infoModalIsOpen`  is depands on the localsotrage `first-time` item. Default is true so when user open the app in first visit will show the instruction. While user close the instruction, the  `first-time` item will set to false at the same time, so it would't show at the next visit

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -328,13 +333,13 @@ function App() {
       />
       <Modal
         isOpen={infoModalIsOpen}
-        onRequestClose={() => setInfoModalIsOpen(false)}
+        onRequestClose={() => handleInfoClose()}
```
Comment (octokatherine) [inline src/App.js:336]:
```suggestion
        onRequestClose={handleInfoClose}
```

File: src/App.js [hunk 2]
```diff
@@ -328,13 +333,13 @@ function App() {
...
         style={customStyles}
         contentLabel="Game Info Modal"
       >
         <button
           className="absolute top-4 right-4 rounded-full nm-flat-background text-primary p-1 w-6 h-6 sm:p-2 sm:h-8 sm:w-8"
-          onClick={() => setInfoModalIsOpen(false)}
+          onClick={() => handleInfoClose()}
```
Comment (octokatherine) [inline src/App.js:342]:
```suggestion
          onClick={handleInfoClose}
```

**Review by octokatherine — CHANGES_REQUESTED** [review]
A couple suggested changes but this is great, thank you!

**Comment by wilsonych** [comment]
Thanks for your suggestion

**Review by octokatherine — APPROVED** [review]

**Pull request merged**